README.md

```diff
@@ -184,7 +184,7 @@ Run frontend tests:
 
 ### Integration Tests
 
-Integration tests are run in a headless Chrome driven by [protractor](http://www.protractortest.org/#/).  Requirements include Chrome, a working cluster, kubectl, and bridge itself (see building above).
+Integration tests are run in a headless Chrome driven by [protractor](http://www.protractortest.org/#/). Requirements include Chrome, a working cluster, kubectl, and bridge itself (see building above).
```

Comment:
Google Chrome/Chromium >= 60 is listed as a dependency at the top of the README. We should update that to say 76.

Reply:
found it :+1: